src/bikeracks/__init__.py

```diff
@@ -75,8 +78,33 @@ def submitFeedback():
                 # writing the data rows 
                 csvwriter.writerow(row)
         except Exception as e:
-            return (e, 500)
+            return (str(e), 500)
         return ('OK', 200)
+
+    # Function for processing "suggest removal" request
+    @app.route('/submitRemovalSuggestion', methods=('POST',))
+    def submitSuggestion():
+        rack_id = request.form.get("rack_id", type=float)
+        removal_reason = request.form.get("removal_reason", type=str)
```

Comment:
I thought this was going to be an enum?

An integer value that corresponds to a reason?